src/LibraryManager.Vsix/Commands/InstallLibraryCommand.cs

```diff
@@ -88,7 +93,7 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
                 string configFilePath = Path.Combine(rootFolder, Constants.ConfigFileName);
                 IDependencies dependencies = Dependencies.FromConfigFile(configFilePath);
 
-                Manifest manifest = await Manifest.FromFileAsync(configFilePath, dependencies, CancellationToken.None).ConfigureAwait(false);
```

Comment:
>.ConfigureAwait(false); [](start = 118, length = 23)

Intentionally deleted?

Reply:
I moved it inside GetManifestAsync(..) so I can check if file is open

---
In reply to: [203157458](https://github.com/aspnet/LibraryManager/pull/323#discussion_r203157458) [](ancestors = 203157458)

Reply:
but it seems like it should be here too, no?

---
In reply to: [203157935](https://github.com/aspnet/LibraryManager/pull/323#discussion_r203157935) [](ancestors = 203157935,203157458)

Reply:
Sorry didn't realize that you highlighted ConfigureAwait(..)
Updated in iteration 3.

---
In reply to: [203158277](https://github.com/aspnet/LibraryManager/pull/323#discussion_r203158277) [](ancestors = 203158277,203157935,203157458)

Reply:
Updated with the changes we made offline.